packages/web-components/fast-foundation/src/select/listbox.ts

```diff
@@ -0,0 +1,34 @@
+import { attr, observable } from "@microsoft/fast-element";
+import { keyCodeSpace } from "@microsoft/fast-web-utilities";
+import { FormAssociated } from "../form-associated/index";
+import { customElement } from "@microsoft/fast-element";
+import { FASTElement } from "@microsoft/fast-element";
+
+export class Listbox extends FASTElement {
+    @observable
+    public defaultSlottedNodes: Node[];
+
+    /**
+     * Set to true when the component has constructed
+     */
+    private constructed: boolean = false;
+
+    constructor() {
+        super();
+        this.constructed = true;
+    }
+
+    public keypressHandler = (e: KeyboardEvent): void => {
+        console.log('button pressed');
+        switch (e.keyCode) {
+            case keyCodeSpace:
+                console.log('space pressed');
```

Comment:
Please pull the console.logs :)

Reply:
What should we do in this case? Have we not yet set up our keyboarding behavior of the listbox?